main.js

```diff
@@ -0,0 +1,31 @@
+
+
+const renderApts = function () {
+    $('#results').empty()
+    $.ajax({
+        method: 'GET',
+        url: `https://randomuser.me/api/?results=100`,
+        success: function (response) {
+            $('#results').empty()
```

Comment:
you already have this at the top of the function. You do not need to empty again